app.js

```diff
@@ -0,0 +1,39 @@
+const dotenv = require("dotenv").config();
```

Comment:
```suggestion
require("dotenv").config();
```
require().config()를 하고 있기 때문에 변수 선언도 필요없습니다 :)

Reply:
네 확인했습니다 ! 꿀팁 감사합니다.